components/rich_text_editor/rich_text_editor.vue

```diff
@@ -283,3 +299,13 @@ export default {
   },
 };
 </script>
+
+<style lang="less">
+.ProseMirror p.is-editor-empty:first-child::before {
+    content: attr(data-placeholder);
+    float: left;
+    color: #adb5bd;
```

Comment:
This should most likely use `color: var(--fc-placeholder);` that DtInput uses

Reply:
ty good catch!